### DISCUSSION TITLE
high and low eighth (8ve, 8va) be abbreviated as this in piano score?

### DISCUSSION THREAD

**Discussion by wenshuoge** [CoderLine/alphaTab#1321] [Ideas]
In Musescore
![image](https://github.com/CoderLine/alphaTab/assets/115636025/86c3aac2-d979-4f5a-abf3-f321117b3179)

In AlphaTab
![image](https://github.com/CoderLine/alphaTab/assets/115636025/37d3e1a9-448d-4172-81ac-0baa8753a918)

**Comment by hanghang2** [comment]
I had the whole problem, too

**Comment by Danielku15** [comment]
Please try to phrase out some proper post to discuss your needs. Share details about files and what you are doing and plan to do.
